asset/css/primary-submit-btn-duplicate.less

```diff
@@ -0,0 +1,9 @@
+.primary-submit-btn-duplicate {
```

Comment:
I would add a description here, something like:

> Automatically set CSS class for duplicated submit buttons used for implicit form submission that should be invisible and not take up any space. `display: none` is not an option, because at least Safari will then ignore the element completely.